src/report/repository/report-repository.service.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  ForbiddenException,
+  Injectable,
+  NotFoundException,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { ConfigService } from '@nestjs/config';
+import { catchError, map, Observable } from 'rxjs';
+
+interface ReportEntity {
+  id: string;
+  key: string;
+  value: {
+    rev: string;
+  };
+  doc: ReportDoc;
+}
+
+interface ReportDoc {
+  _id: string;
+  _rev: string;
+  title: string;
+  aggregationDefinitions: any; // TODO better typing
```

Comment:
I think this is at least `string[]`

https://github.com/Aam-Digital/ndb-core/blob/e6b40960babd7d8b89fd4bca554de63e24c2a52c/src/app/features/reporting/report-config.ts#L62-L68